CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-08-27
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240827.
+
+### Announcements
+
+* AKS version 1.27 is now deprecated, onboard to LTS if you still need to operate on 1.27.
+* [Trusted launch for AKS](https://learn.microsoft.com/en-us/azure/aks/use-trusted-launch) is now GA. 
```

Comment:
This would be "Features" under the release notes. Also please add the GA doc link.